lib/v2/screens/authentication/recover/recover_account_search/recover_account_screen.dart

```diff
@@ -70,26 +83,34 @@ class _RecoverAccountScreenState extends State<RecoverAccountScreen> {
                       });
                     },
                   ),
+                  state.isValidAccount == true
```

Comment:
We already know is a bool var.

`state.isValidAccount? Container(`

Reply:
you are right lol, idk why I always do this (fix)